braid/delta.h

```diff
@@ -0,0 +1,75 @@
+/*BHEADER**********************************************************************
+ * Copyright (c) 2013, Lawrence Livermore National Security, LLC.
+ * Produced at the Lawrence Livermore National Laboratory.
+ *
+ * This file is part of XBraid. For support, post issues to the XBraid Github page.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License (as published by the Free Software
+ * Foundation) version 2.1 dated February 1999.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the IMPLIED WARRANTY OF MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the terms and conditions of the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ ***********************************************************************EHEADER*/
+
+/** \file delta.h
+ * \brief Define internal XBraid headers for Delta correction.
+ *
+ * This file contains the internal XBriad headers for Delta correction,
+ */
+
+#ifndef _braid_DELTA_HEADER
+#define _braid_DELTA_HEADER
+
+#include "_braid.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
```

Comment:
For consistency with the other header files, 
- There shouldn't be an indent
- The function description should be formatted like
/**
 * Function description
 * ...
 **/